### PR TITLE
Update Recommendation 3 to avoid ambiguity of roles

### PR DESCRIPTION
**Pull request by malvikasharan**
In recommendation 3, the roles of Counselors and Ombudspersons have been blurred. We already have established roles for the ombudsperson who are external to the community, and recommendation 3 should highlight that we should have counsellors from within the community to facilitate CoC work. 

These points have been raised in the CoC facilitators draft by me and Karin Lagesen and suggested by @serahrono I am opening this PR for @kariljordan's review.

### CODE REVIEW

File: 2019/incidents-outside-cocc/2019-09-19-cocc-taskforce-summary-recommendations.md [hunk 1]
```diff
@@ -32,11 +32,11 @@ We recommend leaving the Code of Conduct Committee as an __enforcement mechanism
...
 
-__RATIONALE:__ Only one of the 12 anonymous feedback responses received during our three week feedback collection period dealt with an incident that was shared with the CoCc. It could thus be argued that there might be many unreported incidents that simply go unnoticed. The objective of the Code of Conduct, and the mandate of the CoC committee as its enforcer, should be not only to enforce punishments for violations, but to provide a safe space where everyone can share her/his perceived abuse. Filtering of incidents in light of CoC guidelines would then identify which incidents constitute violations of the CoC. To this end the ombudspersons, external to the CoC committee, would: favor sharing incidents by separating hearing and counseling (ombudsperson) versus evaluation and enforcement (CoC Committee); and provide a filtering stage that would prevent overloading the CoC committee.
```
Comment (kariljordan) [inline 2019/incidents-outside-cocc/2019-09-19-cocc-taskforce-summary-recommendations.md:39]: Thank you for catching this!

**Comment by kariljordan** [comment]
Hi @malvikasharan. Thanks so much for this PR. As this report includes __recommendations__, I caution against changing the content of the report. The Community Development Team and CoCc have developed the CoC Facilitator module as a result of the recommendation that came out of this report, so I don't think we need to update the terminology that was used in their recommendations. The CoC facilitators are serving as counselors, and that in essence is what the task force was recommending. If the terminology in the CFP module is correct I think that's what is most important. We could even point to this recommendation report and say that the task force recommended Ombudspersons, however we already have established roles for the ombudsperson who are external to the community.

The other task force members were Samantha Ahern, Karen Cranston, Christopher Felker, and Luca Di Stasio if you'd like to get their take on revising the official recommendation report.

**Comment by malvikasharan** [comment]
Thanks Kari. I had suggested in the document to indicate that we are amending/editing this section to avoid confusions around the roles. I don't need to edit this document as I agree that this is approved and is in history.

@serahrono Can you please have a look at Kari's comment and we can see what we can do in the document?

**Comment by serahrono** [comment]
thanks @kariljordan and @malvikasharan, on it!

**Pull request closed without merging**